docs/common-coreboot-docs/custom_logo.md

```diff
@@ -24,35 +24,40 @@ and `cbfstool`.
 
 1. Obtain cbfstool source code (skip if you've already cloned the coreboot
    source):
-	```bash
-	$ git clone https://review.coreboot.org/coreboot.git
-	```
+
+    ```bash
+    $ git clone https://review.coreboot.org/coreboot.git
+    ```
```

Comment:
@mkopec please use correct indentation and surrounding of code blocks. Please make sure you using pre-commit before pushing changes.